cmd/relayproxy/api/middleware/zap.go

```diff
@@ -65,9 +66,9 @@ func ZapLogger(log *zap.Logger, config *config.Config) echo.MiddlewareFunc {
 				log.With(zap.Error(v.Error)).Error("Server error", fields...)
 			case n >= 400:
 				log.With(zap.Error(v.Error)).Warn("Client error", fields...)
-			case n >= 300:
+			case n >= 300 && cfg.VerboseRequestLogging:
 				log.Info("Redirection", fields...)
-			default:
+			case cfg.VerboseRequestLogging:
 				log.Info("Success", fields...)
```

Comment:
We should probably play with the log levels here rather that having to check `VerboseRequestLogging`.

Something like 
```go
case n >= 400:
  log.With(zap.Error(v.Error)).Warn("Client error", fields...)
case n >= 300:
  log.Info("Redirection", fields...)
default:
  log.debug("Success", fields...)

```

Reply:
@thomaspoignant debug won't help either, I want to be able to fully remove those request logs.

Reply:
@thomaspoignant I believe I misunderstood the first comment, playing with the log level will help, but only if we can exclude lower levels. Would that work? I can simply flip the flag as you suggested as `LogLevel` and in the secton of code filter the ones that are lower than specified.

Reply:
Yes sorry if it was not clear, you can tell `uber/zap` to log only regarding to the `LogLevel` configured.

Reply:
@thomaspoignant nice! I'm pushing the change with the LogLevel instead 👍🏻

Reply:
@thomaspoignant I'm sure that what I suggested in the PR is enough, probably have to update something else. Regarding the Debug flag, idk what is the process you use for deprecating fields, but the log level is enough enable verbosity as you proposed.